package/batocera/emulators/sdlpop/001-fix-Makefile.patch

```diff
@@ -0,0 +1,29 @@
+--- a/src/Makefile	2021-01-03 20:23:00.000000000 +0100
```

Comment:
Use cmake and we won't need any patch
https://github.com/NagyD/SDLPoP/blob/master/src/CMakeLists.txt